packages/google_maps_flutter/google_maps_flutter_android/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.4.7
+
+* Update annotation and espresso dependencies.
```

Comment:
Did you update espresso dependencies in this package?

Reply:
Good catch, I used the same message for both packages. Will fix here and in the other package so that each changelog is specific to what was changed.